tests/common/configuration/test_credentials.py

```diff
@@ -286,3 +286,20 @@ def test_run_configuration_slack_credentials(environment: Any) -> None:
     environment["RUNTIME__SLACK_INCOMING_HOOK"] = "DBgAXQFPQVsAAEteXlFRWUoPG0BdHQ-EbAg=="
     c = resolve_configuration(RunConfiguration())
     assert c.slack_incoming_hook == "DBgAXQFPQVsAAEteXlFRWUoPG0BdHQ-EbAg=="
+
+
+def test_aws_credentials_resolved_from_default(environment: Dict[str, str]) -> None:
```

Comment:
please test default credentials in tests/load - I do not want to install any extras for common tests